data_validation/cli_tools.py

```diff
@@ -641,6 +733,71 @@ def _add_common_arguments(parser):
     )
 
 
+def _add_common_partition_arguments(parser, required_arguments=None):
+    """Add all arguments common to get-partition command"""
+
+    # Group all Required Arguments together
+    if required_arguments is None:
+        required_arguments = parser.add_argument_group("required arguments")
+
+    required_arguments.add_argument(
+        "--source-conn", "-sc", required=True, help="Source connection name"
+    )
+    required_arguments.add_argument(
+        "--target-conn", "-tc", required=True, help="Target connection name"
+    )
+    required_arguments.add_argument(
+        "--config-dir",
+        "-cdir",
+        required=True,
+        help="Directory Path to store YAML Config Files",
+    )
+    required_arguments.add_argument(
```

Comment:
I think we should get rid of this flag and instead just support a --partition-key flag. This way the partition key and PK don't necessarily need to be the same column (if no --partition-key is supplied, we can use the PK). In the future when we support hashmod or partitionkeymod we can add flags depending on what user input we expect.

Reply:
Done. Removed `--partition-type` flag. Default partitions would be generated on `--partition-key`. 
`--partition-key` has been made optional, hence in its absence, it will default to first `--primary-keys`